### PR TITLE
EWPP-772: Remove HTML markup in the contact form emails.

### PR DESCRIPTION
**Pull request by georgebaev**
### Description

Removes the HTML markup in the contact form emails

### Change log

- Added: Instructions for installing MailDev
- Fixed: HTML markup removed

### CODE REVIEW

**Review by sergepavle — APPROVED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by upchuk — APPROVED** [review]

File: tests/src/FunctionalJavascript/MessageFormTest.php [hunk 1]
```diff
@@ -280,4 +281,80 @@ public function testDefaultFormNotChanged(): void {
...
+   */
+  protected function assertTestEmailBodies(array $captured_emails): void {
+    // First email is the outgoing one.
+    $expected = <<<EOF
+tester (not verified) (tester@example.com) sent a message using the contact
+form at http://web:8080/build/contact/oe_contact_form.
+
+
+
+      The sender's name
```
Comment (drishu) [inline tests/src/FunctionalJavascript/MessageFormTest.php:300]: I see the issues in drone with composer lowest, would it make sense to test for partial output with strpos like above ?

File: tests/src/FunctionalJavascript/MessageFormTest.php [hunk 2]
```diff
@@ -190,6 +190,7 @@ public function testCorporateForm(): void {
     $this->assertTrue(strpos($captured_emails[1]['body'], 'Test subject') !== FALSE);
```
Comment (drishu) [inline tests/src/FunctionalJavascript/MessageFormTest.php:190]: I think these are redundant with the new assertTestEmailBodies as $captured_emails[1]['body'] are already tested there

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/src/FunctionalJavascript/MessageFormTest.php (hunk 1 above)
Comment (sergepavle) [reply tests/src/FunctionalJavascript/MessageFormTest.php:300]: Failure on lowest related to the difference in rendering between <=8.9.10 and >=8.9.11 version of core (and particular this change https://git.drupalcode.org/project/drupal/commit/05083ea0824377a2b41613671cf5a00b3858f696).
For making this green for both versions - it makes sense to normalize compared strings by removing more than 1 spaces, newline symbols, etc., or use the suggestion of Andras.

Comment (upchuk) [reply tests/src/FunctionalJavascript/MessageFormTest.php:296]: Thanks @sergepavle for finding this

**Review by sergepavle — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**